src/procurementUnit/ProcurementUnitItem.tsx

```diff
@@ -75,6 +78,24 @@ const ProcurementUnitItem: React.FC<PropTypes> = observer(
       ? procurementUnit?.area?.name
       : OperatingAreaName.Unknown
 
+    // The logic behind finding displayed contractUnit: selecting the latest contract that is currently valid
+    // meaning if there are multiple contracts being valid, we select the one with the latest startDate
+    let displayedContractUnit: Contract | null = null
+    if (currentContracts && currentContracts.length > 0) {
+      displayedContractUnit = currentContracts
```

Comment:
You could probably just do:

```suggestion
    let displayedContractUnit = currentContracts
```

And have it be undefined (instead of null) if no contract was found. Same outcome.

Reply:
Using Conditional ? operator now